consensus/service/src/validators.rs

```diff
@@ -175,41 +154,12 @@ impl<L: Ledger + Sync> TxManagerUntrustedInterfaces for DefaultTxManagerUntruste
 #[cfg(test)]
 pub mod well_formed_tests {
     use super::*;
-    use mc_common::logger::{bench_with_logger, test_with_logger, Logger};
-    use mc_ledger_db::LedgerDB;
-    use mc_transaction_core::{
-        constants::MAX_TOMBSTONE_BLOCKS, ring_signature::KeyImage, tx::Tx,
-        validation::TransactionValidationError,
-    };
+    use mc_common::logger::{test_with_logger, Logger};
     use mc_transaction_core_test_utils::{
         create_ledger, create_transaction, initialize_ledger, AccountKey,
     };
     use rand::SeedableRng;
     use rand_hc::Hc128Rng;
-    use test::Bencher;
-
-    fn is_well_formed(tx: &Tx, ledger: &LedgerDB) -> TransactionValidationResult<()> {
```

Comment:
This function turned out to be really misleading because it performs both the untrusted and enclave parts of the well-formed check, but the unit tests here should only exercise the untrusted part.